torch/csrc/jit/python/python_tree_views.cpp

```diff
@@ -162,7 +162,18 @@ void initTreeViewBindings(PyObject* module) {
                        const Def& getter,
                        Def* setter) {
         return Property::create(r, name, getter, wrap_maybe(r, setter));
-      }));
+      }))
+      .def("name", [](const Property& property) { return property.name(); })
+      .def(
+          "getter_name",
+          [](const Property& property) { return property.getter().name(); })
+      .def("setter_name", [](const Property& property) {
+        if (property.setter().present()) {
+          return property.setter().get().name();
+        }
+
+        return Ident::create(property.range(), "");
```

Comment:
Can we use None here instead ? empty string as null value could lead to some confusing debugging

Reply:
`None` as a string literal? I tried to use `py::none()` but that didn't work because of what I think is a problem inferring the return type of this lambda. Is there an example you can point me to?

Reply:
you can return None with c10::optional, and cast both returns to `c10::optional<Ident>` so that the return type gets properly inferred

Reply:
Fixed.